app/src/main/res/layout-sw600dp-port/topic_revision_summary_view.xml

```diff
@@ -28,14 +28,15 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content">
 
-      <ImageView
+      <org.oppia.app.customview.LessonThumbnailImageView
```

Comment:
we can reformat this file.

Reply:
Reformatted all 4 files of `topic_revision_summary_view.xml`